asv_bench/benchmarks/benchmarks.py

```diff
@@ -148,6 +193,38 @@ def time_merge(self, shapes, how, sort):
         )
 
 
+class TimeMerge2:
+    param_names = ["shapes", "sort"]
+    params = [
+        get_benchmark_shapes("TimeMerge2"),
+        [True, False],
+    ]
+
+    def setup(self, shapes, sort):
+        self.df1 = generate_dataframe("int", *shapes[0], RAND_LOW, RAND_HIGH)
+        self.df2 = generate_dataframe("int", *shapes[1], RAND_LOW, RAND_HIGH)
+        self.rows_count = int(0.25 * shapes[1][0])
+
+    def time_merge_2intkey(self, shapes, sort):
+        execute(IMPL.merge(self.df1, self.df2, sort=sort))
+
+    def time_merge_dataframe_empty_right(self, shapes, sort):
+        execute(IMPL.merge(self.df1, self.df2.iloc[:0], sort=sort))
+
+    def time_merge_dataframe_empty_left(self, shapes, sort):
+        execute(IMPL.merge(self.df1.iloc[:0], self.df2, sort=sort))
+
+    def time_merge_dataframes_cross(self, shapes, sort):
+        execute(
+            IMPL.merge(
+                self.df1.loc[: self.rows_count],
+                self.df2.loc[: self.rows_count],
+                how="cross",
```

Comment:
We are defaulting to pandas for `cross` value. Do we want to measure this?

Reply:
I think it's time to add such cases.

Reply:
Why? It is just an additional plot in ASV results, where we will see a slower Modin operation. I think we can add this later when we implement the case.

Reply:
I see that you removed the test but left the setup for it. Let's remove the setup as well.